src/main/java/com/epam/spring/homework2/beans/BeanB.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.spring.homework2.beans;
+
+import com.epam.spring.homework2.annotations.Validation;
+import org.springframework.beans.factory.annotation.Value;
+
+@Validation
+public class BeanB {
+    @Value("${beanB.name}")
+    private String name;
+    @Value("${beanB.value}")
+    private int value;
+
+    @Override
+    public String toString() {
+        return "BeanB{" +
+                "name='" + name + '\'' +
+                ", value='" + value + '\'' +
+                '}';
+    }
+
+    private void initMethodBeanB() {
```

Comment:
I know it's not causing errors, but please update access modifier for all init and destroy methods, they should be `public`